### ISSUE TITLE
fullscreen slideshows on multiple monitors

### ISSUE THREAD

**Issue by clintrowe** [derf/feh#447]
I'm trying to run separate fullscreen slideshows on multiple monitors using both -F and --geometry flags as below for 2 monitors.  Unfortunately, both slideshows open at +0+0

feh -F --geometry 1920x1080+0+0 <first-list-of-images> &
feh -F --geometry 1920x1080+1920+0 <second-list-of-images>

Am I approaching this incorrectly? Is this possible?

**Comment by trapd00r** [comment]
Yes, I can confirm it works perfectly on 3 monitors using ratpoison.

**Comment by clintrowe** [comment]
I'm using GDM and neither --no-xinerama nor --xinerama-index makes any difference, the window always opens at +0+0.  I poked around in the code a little and quickly found that if I hard-coded a value for dx = 960 in winwidget_render_image, I could get it to display half of my image in the window locked to +0+0.  I tried something similar by hard-coding a value for x in XCreateWindow, but that didn't change anything, either.

**Comment by clintrowe** [comment]
I have a workaround that I can use.  I start the slideshow on display 1 in fullscreen (-F) and start the others with the geometry offset and borderless (-x). This allows the slideshow on display 1 to cover over the top and bottom menu/status bars GDM insists on.  In the end, they all look the same.

Thanks for your input.
